docs/sources/github.md

```diff
@@ -0,0 +1,47 @@
+# Event Source for Github
+
+This event source creates a webhook to listen for incoming Github Events, turning received requests into [CloudEvents][ce] to be consumed by other TriggerMesh components.
+
+## Prerequisite(s)
+
+- GitHub Tokens
```

Comment:
Could you add some additional details about whether this should be an oAuth token or personal access token.

Reply:
In the block below it is elaborated.